### ISSUE TITLE
Restoring wallet using seed phrase does not import all created accounts and funds/collectibles

### ISSUE THREAD

**Issue by bawse** [Berry-Pool/nami-wallet#192]
Hi,

I have a screenshot of my seed phrase and I'm trying to transfer my Nami wallet onto another laptop. I transferred some ADA and minted an NFT on a new account on the same wallet, before attempting to import onto another laptop. 

This is the address of the original wallet which holds my nft and some ada:
addr1q9cmkg9wan53ywr2k5vww8zu2kegalg75r3k84a6k2zk782ev4dxzz35z2sg28qakxjn2vxxxpez7x3yesy8tedkaqtqu638k0

After reimporting onto another laptop, I can only see one account and my funds/nft is not available. The wallet address after reimport is:
addr1q969jzt5qcx6sjjqh2xh0f76w9ffzqh9ves3h8aq93ll0ju40dk7ersxhzssx6x0w3q7drc0fal05q39spyqt6zcq0kqaurku7

Is it possible to recover the account or show the other "account" that I created and minted my NFT into?

It seems like this issue has been raised by many people before. Any help is appreciated.

**Comment by alessandrokonrad** [comment]
Is it possible that you created another acount in Nami and never used the first one? If so try to create a new account. The account order is always the same. Nami tries to find existing accounts you have automatically, but if there is a gap in the accounts and one is totatlly unused it will stop looking further.

**Comment by bawse** [comment]
@alessandrokonrad yes it's likely that I created a second account, and the first one was unused. My assets were in the second account from what I can remember. 

I just tried creating another account, but it still doesn't show the assets as expected. Checking the wallet address on cardanoscan shows the assets are present, but I can't see anything on Nami.

**Comment by alessandrokonrad** [comment]
Okay so account 2 has the correct address, but doesn't show you the balance?

**Comment by bawse** [comment]
@alessandrokonrad sorry I should have mentioned in my reply that none of the accounts I create have the same address as the wallet which hold my assets. I'm absolutely certain that the seed phrase is correct too.

**Comment by aidansmithh** [comment]
this has happened to me. I got a new laptop, installed nami and clicked import wallet, put in my seed phrases and it created a new wallet?? It wasn't showing any of my assets now I can't get into my wallet with the assets in
